cmd/ocsp-updater/main.go

```diff
@@ -120,11 +131,63 @@ func newUpdater(
 		})
 	}
 
-	updater.ocspMinTimeToExpiry = config.OCSPMinTimeToExpiry.Duration
+	if config.AkamaiBaseURL != "" {
+		issuer, err := core.LoadCert(issuerPath)
+		ccu, err := akamai.NewCachePurgeClient(
+			config.AkamaiBaseURL,
+			config.AkamaiClientToken,
+			config.AkamaiClientSecret,
+			config.AkamaiAccessToken,
+			config.AkamaiPurgeRetries,
+			config.AkamaiPurgeRetryBackoff.Duration,
+			log,
+			stats,
+		)
+		if err != nil {
+			return nil, err
+		}
+		updater.ccu = ccu
+		updater.issuer = issuer
+	}
 
 	return &updater, nil
 }
 
+// sendPurge should only be called as a Goroutine as it will block until the purge
+// request is successful
+func (updater *OCSPUpdater) sendPurge(der []byte) {
+	cert, err := x509.ParseCertificate(der)
+	if err != nil {
+		updater.log.AuditErr(fmt.Errorf("Failed to parse certificate for cache purge"))
+		return
+	}
+
+	req, err := ocsp.CreateRequest(cert, updater.issuer, nil)
+	if err != nil {
+		updater.log.AuditErr(fmt.Errorf("Failed to create OCSP request for cache purge"))
+		return
+	}
+
+	// Create a GET style OCSP url for each endpoint in cert.OCSPServer (Akamai
+	// translates OCSP POST requests to GETs so we end up purging both as a side
```

Comment:
Does this mean https://github.com/letsencrypt/boulder/issues/996 can be resolved? If you have any extra detail or supporting documentation here it would be great to link to it. Is it just a matter of our specific config? If so, what's the name of the config in the control center?